examples/kv/builder_template.py

```diff
@@ -23,6 +23,6 @@ def build(self):
             root.add_widget(wid)
         return root
 
+
 if __name__ == '__main__':
     BlehApp().run()
-
```

Comment:
"./examples/kv/builder_template.py:29:1: W391 blank line at end of file
"
i encountered this so i made changes. :)